bot/cogs/weather_cog.py

```diff
@@ -0,0 +1,353 @@
+# Thomas Delvaux
+# 12-16-2020
+
+import logging
+
+import discord
+import discord.ext.commands as commands
+
+import bot.extensions as ext
+from bot.messaging.events import Events
+from bot.bot_secrets import BotSecrets
+from bot.consts import Colors
+
+import aiohttp
+import asyncio
+import re
+import datetime as dt
+
+log = logging.getLogger(__name__)
+url = "https://api.openweathermap.org/data/2.5/onecall"
```

Comment:
Module constants are always ALL_CAPS_SNAKE_CASE and if you see anywhere else they arent let me know so i can fix my mess up :grin: